infra/gcp/clusters/projects/artifact-management-sandbox/provider.tf

```diff
@@ -0,0 +1,25 @@
+/*
+This file defines:
+- Required provider versions
+- Storage backend details
+*/
+
+terraform {
+
+  backend "gcs" {
+    bucket = "k8s-infra-clusters-terraform"
+    prefix = "artifact-management-sandbox"
```

Comment:
@spiffxp will appear in the audit report. Created by `terraform init`.

Reply:
ack, I will try to capture this as part of https://github.com/kubernetes/k8s.io/pull/1952